lib/ui/shared/smart_widgets/expandable_textfield/expandable_textfield_screen.dart

```diff
@@ -14,19 +14,21 @@ class ExpandableTextFieldScreen extends HookWidget {
     required this.widget,
     required this.sendMessage,
     required this.hintText,
+    required this.textController
   }) : super(key: key);
   final Widget widget;
   final Function(String message) sendMessage;
   final String hintText;
   final focusNode = FocusNode();
+  final TextEditingController textController;
+
   final keyboardVisibilityController = KeyboardVisibilityController();
   Stream<bool> get stream =>
       keyboardVisibilityController.onChange.asBroadcastStream();
 
   @override
   Widget build(BuildContext context) {
     double maxSize = MediaQuery.of(context).size.height - kToolbarHeight - 30;
-    TextEditingController textController = useTextEditingController();
     return ViewModelBuilder<ExpandableTextFieldScreenViewModel>.reactive(
```

Comment:
Viewmodel should be with the build context and not inside

Reply:
I didn't get you really? but then, The file was never within my scope, all I did there was to remove the textEditing controller and pass in another controller from its parent widget for my implementation

Reply:
Resolved